app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationHelper.java

```diff
@@ -268,6 +270,7 @@ public static void make(final Context context, Notification body, AccountEntity
     private static NotificationCompat.Builder newNotification(Context context, Notification body, AccountEntity account, boolean summary) {
         Intent summaryResultIntent = new Intent(context, MainActivity.class);
         summaryResultIntent.putExtra(ACCOUNT_ID, account.getId());
+        summaryResultIntent.putExtra(TYPE, body.getType().name());
```

Comment:
There's a small risk of using `name` here if the enum name ever changes across versions. It looks like this particular class is excluded from proguard obfuscation so it should be safe but open to suggestions if there's a concern.